arm-advisor/2017-03-31/swagger/advisor.json

```diff
@@ -0,0 +1,679 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"version": "2017-03-31",
+		"title": "AdvisorManagementClient",
+		"description": "REST APIs for Azure Advisor"
+	},
+	"host": "management.azure.com",
+	"schemes": [
+		"https"
+	],
+	"consumes": [
+		"application/json"
+	],
+	"produces": [
+		"application/json"
+	],
+	"paths": {
+		"/subscriptions/{subscriptionId}/providers/Microsoft.Advisor/generateRecommendations": {
+			"post": {
+				"tags": [
+					"GenerateRecommendations"
+				],
+				"description": "Initiates the recommendation generation or computation process for a subscription. This operation is asynchronous. The generated recommendations are stored in a cache in the Advisor service.",
+				"operationId": "Recommendations_Generate",
+				"parameters": [
+					{
+						"$ref": "#/parameters/subscriptionIdParameter"
+					},
+					{
+						"$ref": "#/parameters/apiVersionParameter"
+					}
+				],
+				"responses": {
+					"202": {
```

Comment:
Here you are giving the status 202 which is accepted. But what is the final status? 200? If so, please include it